Documentation addition for viewer.world

alanaberdeen opened the pull request:
In response to #1477

The changes in `viewer.js` were simple to find. But the equivalent documentation edits for `source` in `tiledimage.js` are a little hidden, is the `.source` member being added when merging `options` object in? If so, where should the jsdoc comments sit?

See: 
https://github.com/openseadragon/openseadragon/blob/cd0a4aee16c0435b4d4c4e98807c0d14e716d707/src/tiledimage.js#L177

iangilman commented:
Good question! It actually doesn't matter that much where the comments are; they'll still get picked up. Probably at the top of the TiledImage constructor would be good:

https://github.com/openseadragon/openseadragon/blob/master/src/tiledimage.js#L88

alanaberdeen commented:
Ah I see, thought there might be an internal styling guide about where to put the comments I should adhere to. Added the `TiledImage.source` docs in.

Also noticed in, `viewer.js` that `source` could also be accessed directly from the viewer instance... added a comment to document that but it might need detailing further. One question I have is if we have multiple images then which one would `viewer.source` default to?

iangilman commented:
Yes, I appreciate your asking whether we had a preferred location! 

The `source` in `viewer.js` is just there for backwards compatibility, and not an official part of the API, so I would prefer to leave it undocumented. I believe it's the first TileSource in the World, but people shouldn't rely on it.

alanaberdeen commented:
Ah I see, fair enough. I've added a general comment to note that in the code but of course edit as you wish before any merges!
